18/README.md

```diff
@@ -2,211 +2,158 @@
+At this point the Election starts. Independently, and asynchronously, each Validator can spend its **Vote Tokens** to an **Election Address** to show agreement on the matter of change. The Election Address is the `id` of the first `CREATE` transaction. Once the Vote tokens has been transferred to that address, it is not possible to transfer it again, because their private key is not known.
 
-During the `end_block` call, all transactions about to be committed are checked. Every transfer of a vote token triggers a functions that counts the number of positive votes of Election over the number of voters. If the ratio is greater than ⅔, then the current validator commits the change. Given the BFT nature of the system, all non-Byzantine Validator will commit the change at the same block height.
+During the `end_block` call, all transactions about to be committed are checked. Every transfer of a vote token triggers a functions that counts the number of positive votes of Election over the number of voters. If the ratio is greater than ⅔, then the current Validator commits the change. Given the BFT nature of the system, all non-Byzantine Validator will commit the change at the same block height.
 
 Each Validator checks every new transaction that is about to be committed in a block. The process is roughly the following:
 1. If the transaction is **not** a valid Vote, return.
-4. If `asset.data.id` is **not** a valid Election, return.
-2. If the Election has less than ⅔ of positive votes, return.
-3. Execute the logic to implement the Election.
-
-###  Validate Vote
-A Validator must be able to discern valid Votes from invalid ones. The process is roughly the following:
-
-1. If the transaction is **not** a `TRANSFER`, return false.
-2. If `metadata.type` is **not** `vote`, return false.
-3. If `inputs.owners_before` is **not** a Validator, return false.
-5. Return true.
-
-### Validate Election
-A Validator must be able to discern valid Elections from invalid ones. The process is roughly the following:
-1. `type`: must contain the string `election`.
-2. `name`: name of the election.
-3. `version`: version number.
-4. `matter`: a human readable, short paragraph on the matter of change.
-5. If `inputs.owners_before` is **not** a Validator, return.
-6. If every `outputs.owners_after.amount` is **not** exactly `1`, return.
-7. If `outputs.owners_after` does **not** contain exactly all Validators, return.
-8. Apply any other validation rule enforced by the code specific to the election itself, return false if it fails.
-9. Return true.
+2. If `asset.data.id` is **not** a valid Election, return.
+3. If the Election has less than ⅔ of positive votes, return.
+4. If the Election has been already implemented, return.
```

Comment:
What do you think about replacing `has been already implemented` with `already has more than ⅔ of positive votes`, for clarity?

It would then expand into 3 bullet points instead of 2:
```
- If the Election including the current vote has less than ⅔ of positive votes, return.
- If the Election excluding the current vote has more than ⅔ of positive votes, return. It has been concluded already.
- If the Election excluding the current vote has less than ⅔ of positive votes and including the current vote has more than ⅔ of positive votes, conclude the election.
```